rules/cross-platform/initial_access_zoom_meeting_with_no_passcode.toml

```diff
@@ -38,8 +37,8 @@ tags = [
 type = "query"
 
 query = '''
-event.type:creation and event.module:zoom and event.dataset:zoom.webhook
- and event.action:meeting.created and not zoom.meeting.password:*
+event.type:creation and event.module:zoom and event.dataset:zoom.webhook and
+  event.action:meeting.created and not zoom.meeting.password:*
```

Comment:
is there a chance this field is accidentally omitted?
might not be anything we can do about it

Reply:
it is definitely there https://github.com/P1llus/beats/blob/a3db786ab84e7abdb3b46962c848c6dca5e57fd4/x-pack/filebeat/module/zoom/webhook/_meta/fields.yml#L144-L147

just need 7.10 to be released and then we can refresh the beats schemas

Reply:
i just mean that we're assuming that if the field is not populated, then there is _definitely_ not a password for the meeting. but there's also a chance the field isn't populated because of some other telemetry issue that happens and i accidentally was dropped.

we're okay to move forward, but it could be nice if beats provided a guarantee, like a separate field to say "there was _definitely_ not a password for the meeting"

Reply:
ahhh, yea that is a good point - will add a note to review this logic to further validate